server/scripts/seed.ts

```diff
@@ -161,6 +162,119 @@ const createDonationForm = (requestGroup = null) => {
     });
 };
 
+const createDonorHomepage = () => {
+    const carouselInterval = 10;
+    const numMapTestimonials = faker.datatype.number({
+        'min': 1,
+        'max': 8
+    });
+    const numStats = 3;
+    const map = {
+        defaultMarkerSize: "53px",
+        markerSizes: ["80px", "75px", "72px", "70px", "65px", "61px", "55px", "53px"],
+        points: [
```

Comment:
it seems like each marker has a point and a size, so it might be better to adjust the schema so that the map object stores an array of Marker objects, where each marker has a point object and a size string - instead of storing the data in two separate arrays

Reply:
Good point! I'll make that adjustment :)